test/api/liskApi.js

```diff
@@ -122,14 +123,123 @@ describe('Lisk API module', () => {
 				LSK = new LiskAPI({ testnet: true });
 			});
 
-			it('should set the port to 7000', () => {
+			it('should set the port to 7000 on initialization with option testnet true passed', () => {
 				(LSK).should.have.property('port').be.equal(testPort);
 			});
 
-			it('should set testnet to true', () => {
+			it('should set testnet to true on initialization with option testnet true passed', () => {
 				(LSK).should.have.property('testnet').be.equal(true);
 			});
 		});
+
+		describe('on initialize SSL', () => {
```

Comment:
Maybe have an `initialization` `describe` block, and nest `SSL`, `defaultPeers`, etc inside?